src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/config/TestUtils.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.config;
+
+import java.time.LocalDateTime;
+
+public final class TestUtils {
+
+    private TestUtils() {
+
+    }
+
+    //In some situations a timestamp as a String is compared to another one.
+    //Some timestamps do not have any trailing zeros but others do, so you get
+    //an Assertion error because "54930" milliseconds does not equal "5493" even
+    //though from a timestamp persepective these two values are the same
+
+    public static String removeTrailingZeroes(LocalDateTime ts) {
```

Comment:
I don't agree with this method of adjusting the expected value - we need to fix the issue at source - where is the value being truncated ?

The issue is with the serialization/deserialization of the evetnTimestamp field in `EventInformation.java` class

If we modify it as follows

    ```
@JsonSerialize(using = LocalDateTimeSerializer.class)
@JsonDeserialize(using = LocalDateTimeDeserializer.class)
 @JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSSSS")
 private final LocalDateTime eventTimeStamp;
```

then it fixes the root problem and this (untested) TestUtils class can be removed